cas/incident-reporting.md

```diff
@@ -0,0 +1,101 @@
+# Incidents
+
+Incidents can happen. Things do not always go as planned, and that can be okay. However, when incidents occur, the underlying issues (i.e. root causes) must be identified and remediated to discourage the incident from occurring again. Formally documenting the incident in a report encourages an understanding of all contributing root causes, and it presents the opportunity to clearly communicate a remediation plan to reduce the probability of recurrence. 
```

Comment:
I don't think we should describe anything leading to an incident as "okay". I understand the motivation to not deter incident reporting by making them seems overly damaging to a CA, but I would rather see language like

"Things do not always go as planned, and an incident on its own does not generally indicate serious problems with a CA."

Reply:
This is now overcome by events, but we feel the updates made in #187 align with the recommendations included in this comment.